mslib/msui/mscolab.py

```diff
@@ -1412,7 +1412,7 @@ def render_new_permission(self, op_id, u_id):
                 operation_desc = f'{operation["path"]} - {operation["access_level"]}'
                 widgetItem = QtWidgets.QListWidgetItem(operation_desc, parent=self.ui.listOperationsMSC)
                 widgetItem.op_id = operation["op_id"]
-                widgetItem.catgegory = operation["category"]
+                widgetItem.operation_category = operation["category"]
```

Comment:
here and also in stable I wonder what it means that we don't have seen a failure popping up

Reply:
Apparently, you can assign whatever you want as an attribute (yeah, duck-typing!)